examples/ex14.cpp

```diff
@@ -75,6 +90,14 @@ int main(int argc, char *argv[])
    {
       kappa = (order+1)*(order+1);
    }
+   if (rbf_h < order + 1)
+   {
+      rbf_h = std::max(2.01, 1.01 + order);
```

Comment:
This will also be avoided with good default parameters.

Reply:
I removed this.